lib/src/cli/atlas_apps/baas_client.dart

```diff
@@ -154,7 +154,7 @@ class BaasClient {
   }
 
   Future<BaasApp> _createApp(String name, {String? confirmationType}) async {
-    print('Creating app $name ($name$_appSuffix)');
+    print('Creating app $name$_appSuffix');
```

Comment:
yeah definitely need better app names if we are outputting that

Reply:
We have length limitation.